charts/metal3-deploy/0.1.0/charts/baremetal-operator/templates/configmap-ironic.yaml

```diff
@@ -14,12 +14,14 @@ data:
   CACHEURL: "http://{{ $ironicCacheHost }}/images"
   DEPLOY_KERNEL_URL: "http://{{ $ironicBootHost }}/images/ironic-python-agent.kernel"
   DEPLOY_RAMDISK_URL: "http://{{ $ironicBootHost }}/images/ironic-python-agent.initramfs"
-  HTTP_PORT: {{ .httpPort | quote }}
+  HTTP_PORT: {{ .Values.baremetaloperator.httpPort | quote }}
```

Comment:
I don't think this variable is used by the baremetal-operator, it's only used in the ironic pod (the BMO connects to Ironic via the ENDPOINT variables ref [upstream docs](https://github.com/metal3-io/baremetal-operator/blob/main/docs/configuration.md)) - so I think we can probably remove this?

Reply:
No, it does not seem to be used. Property has been removed and a successful deployment of the chart without this property has been made.